react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+
+export default function ToggleSwitch() {
+
+  const [off, on] = useState(false);
+
+  return (
+    <>
+      <label htmlFor="switch" className='toggle'>
+        <input type="checkbox" id="switch"
+        className="toggle-checkbox" onClick={() => {
+          on(!off);
+        }} />
```

Comment:
These arrow functions are a bit more succinct if you leave off the `{`, letting you put it all on one line:
`onClick={() => on(!off)}`